update_chroot

```diff
@@ -97,15 +98,15 @@ EOF
 
 sudo_clobber "/etc/portage/repos.conf/coreos.conf" <<EOF
 [DEFAULT]
-main-repo = portage-stable
+main-repo = gentoo
 
-[gentoo]
-disabled = true
+#[gentoo]
+#disabled = true
 
 [coreos]
 location = ${COREOS_OVERLAY}
 
-[portage-stable]
+[gentoo]
```

Comment:
portage-stable not included our ppc64le patches. I also tested when use portage-stable. It's seems lacks of most packages in ppc64le platform.